pages/common/zint.md

```diff
@@ -0,0 +1,12 @@
+# zint
+
+> CLI tool to generate various (bar/QR) codes.
```

Comment:
Please avoid specifying that the program is used on the command-line.
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#wording
```suggestion
> Generate barcodes and QR codes.
```